XYZDrivers/src/main/java/com/xyzdrivers/models/jdbcDriver.java

```diff
@@ -0,0 +1,435 @@
+/**
+ * @file    jdbcDriver.java
+ * @author  alexander collins
+ * @created 02/11/2017
+ * @notes   - Feel free to fix any stupid mistakes
+ *          - The private functions could probably be made public.
+ *          - Haven't tested it since creating it
+ * @issues  1. getPrimaryKeyName() doesn't work, any function using it won't work.
+ */
+package com.xyzdrivers.models;
+
+import java.sql.*;
+import java.util.*;
+
+public class jdbcDriver
+{
+//variables
+    private Connection DB;
+    private DatabaseMetaData DBMetaData;
+    
+    private PreparedStatement statement;
+    private ResultSetMetaData resultsMetaData;
+    private ResultSet results;
+//constructors
+    /**
+     * Constructor for jdbcDriver.
+     * <code>dbConnection</code> is tested by retrieving <code>DatabaseMetaData</code> from the <code>Connection</code>.
+     * 
+     * @param dbConnection A <code>Connection</code> to the DB being accessed.
+     * 
+     * @throws SQLException 
+     * @throws java.lang.ClassNotFoundException 
+     */
+    public jdbcDriver(Connection dbConnection)
```

Comment:
Keep this constructor and remove the other ones. We will be making the Connection once and just passing it to services as needed.